GridKit/Model/PhasorDynamics/Branch/BranchData.hpp

```diff
@@ -31,18 +32,38 @@ namespace GridKit
     /// Variables able to be monitored for a branch
     enum class BranchMonitorableVariables
     {
-      ir1,
-      ii1,
-      im1,
-      p1,
-      q1,
-      ir2,
-      ii2,
-      im2,
-      p2,
-      q2,
+      IR1,
+      II1,
+      IM1,
+      P1,
+      Q1,
+      IR2,
+      II2,
+      IM2,
+      P2,
+      Q2,
+      SIZE
```

Comment:
Echoing comment by @superwhiskers:

_this pull request leaves casing of enums inconsistent in data structures. for instance, the newly added monitorable variables in busdata are all caps, inconsistent with enums in other files. this should be resolved prior to merge._

Reply:
Updated guidelines provided in #319